qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -121,12 +122,23 @@ def _default_experiment_options(cls) -> Options:
                 used to initialize ``numpy.random.default_rng`` when generating circuits.
                 The ``default_rng`` will be initialized with this seed value everytime
                 :meth:`circuits` is called.
+            gate_error_ratio (Union[str, bool, Dict[str, float]]): The assumption of error ratio
+                of basis gates constituting RB Clifford sequences. When this value is set,
+                the error per gate (EPG) values are computed from the estimated
+                error per Clifford (EPC) parameter in the RB analysis.
+                This value defaults to "default". When explicit gate error ratio is not provided,
+                typical error ratio is provided by :func:`~qiskit_experiments.library.\
+                randomized_benchmarking.rb_utils.lookup_epg_ratio`.
+                The dictionary is keyed on a string label of instruction.
+                Defined instructions should appear in the ``basis_gates`` in the transpile options.
+                If this value is set to ``False``, the computation of EPG values is skipped.
         """
         options = super()._default_experiment_options()
 
         options.lengths = None
         options.num_samples = None
         options.seed = None
+        options.gate_error_ratio = "default"
```

Comment:
I think default here could just be None and then this calculation is skipped unless you actually provide one? Then all handling of calculating a default value can be done in the experiment where the required information is available (backend, basis gates etc)